From db5992a67654d9b0f963084ede34ceb03f142d38 Mon Sep 17 00:00:00 2001
From: Duncan Macleod <duncan.macleod@ligo.org>
Date: Thu, 15 Feb 2024 18:19:08 +0000
Subject: [PATCH 11/12] python: fix cross compiling configuration

when cross-compiling attempting to get Interpreter and Development variables at the same time doesn't work
---
 build/cmake/CondorConfigure.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/build/cmake/CondorConfigure.cmake b/build/cmake/CondorConfigure.cmake
index a1a9c3c659..59f2646c50 100644
--- a/build/cmake/CondorConfigure.cmake
+++ b/build/cmake/CondorConfigure.cmake
@@ -152,7 +152,8 @@ if(NOT WINDOWS)
 	endif(WANT_PYTHON2_BINDINGS AND NOT WANT_PYTHON_WHEELS)
 
 	if (WANT_PYTHON3_BINDINGS AND NOT WANT_PYTHON_WHEELS)
-		find_package (Python3 COMPONENTS Interpreter Development)
+		find_package (Python3 COMPONENTS Interpreter)
+		find_package (Python3 COMPONENTS Development)
 		if (APPLE)
 			# mac doesn't ship a python  interpeter by default
 			# but we want to force the system one, not the one we found
-- 
2.39.5

